esbuild/app.config.js

```diff
@@ -30,7 +30,7 @@ const buildApp = buildConfig => {
  * @returns {Promise}
  */
 module.exports = buildConfig => {
-  process.stderr.write('\u{1b}[1m\u{2728} Building app...\u{1b}[0m\n')
+  process.stdout.write('\u{1b}[1m\u{2728} Building app...\u{1b}[0m\n')
```

Comment:
no big deal really but logging normally goes to stderr

Reply:
Messages logged to `stderr` don't automatically get redirected to output, but shown to the user instead. This is logging output for the script, so should probably be redirected to output rather than treated as error messaging. 

In practice, if I were to pipe running this script into a log file, using `stderr` would add nothing to the log file  (unless explicitly combining error with standard output with `2>&1`). Using `stdout` means the log would contain each of the steps of the build.

No biggie really as you say, and I doubt there's much use case for actually saving logs of the watched build! It just felt like an oversight/typo, as this isn't actually an error message being output. Happy to change back to `stderr` if we insist on this as standard.

Reply:
i don't insist for logging to go back to stderr!
but i'd actually say that if i saw logging going to stdout that _that_ would look like an oversight to me 🤣

Reply:
Fair enough -- I've removed the change to go back to logging to `stderr` by default.

Reply:
🙈 i really wasn't demanding that change! 🙈
but yay, i am hoping that the debounce thingy helps in one of our projects which re/starts really really slowly locally in dev mode